client/components/mobile-back-to-sidebar/index.jsx

```diff
@@ -28,4 +28,4 @@ var MobileBackToSidebar = React.createClass( {
 	}
 } );
 
-module.exports = MobileBackToSidebar;
+module.exports = connect( null, { setLayoutFocus } )( MobileBackToSidebar );
```

Comment:
>I just figured leaving `module.exports` was less risky since it works slightly differently than `export default` if someone uses `require`.

I think this is only true in modules including both named and default exports.